UI/data/locale/en-US.ini

```diff
@@ -193,6 +193,8 @@ Basic.AutoConfig.TestPage.Result.StreamingEncoder="Streaming Encoder"
 Basic.AutoConfig.TestPage.Result.RecordingEncoder="Recording Encoder"
 Basic.AutoConfig.TestPage.Result.Header="The program has determined that these estimated settings are the most ideal for you:"
 Basic.AutoConfig.TestPage.Result.Footer="To use these settings, click Apply Settings. To reconfigure the wizard and try again, click Back. To manually configure settings yourself, click Cancel and open Settings."
+Basic.AutoConfig.Info="The auto-configuration wizard will determine the best settings for streaming and recording based on your computer specs and internet speed."
```

Comment:
As we'll be adding a third option for Virtual Camera, I think we can remove the "for streaming and recording" text.

I also think only the RunAnytime string should be bold.

Reply:
Done.

IMO, it looks better if both are bold.